cppcache/integration-test/ThinClientTransactionsNewXA.hpp

```diff
@@ -0,0 +1,659 @@
+/*
```

Comment:
Maybe you misunderstood my previous ask. This integration test should not be written in the old integration test framework. Please put in the new framework. See cppcache/integration/test for examples.

Reply:
Updated